bin/looker-resources-index/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "looker-resources-index",
+  "description": "A listing of Looker-related resources, and some automation for publishing the listing in consumable formats",
+  "main": "src/resource-data/resources.ts",
```

Comment:
I don't think this will end up being pulled into any of our publish workflows but it would be best to explicitly flag this as a private/non-published package

Reply:
Marked the package as private